bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_5f_ckm_HT-1200to2500_LO_MLM/tt0123j_5f_ckm_HT-1200to2500_LO_MLM_customizecards.dat

```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+set param_card mass 6 172.5
+set param_card mass 25 125.0
```

Comment:
the process is pure qcd, so no qed lines including the higgs. you can remove the line. on the other hand you change the top mass, so you might want to change the top width with auto too. will be done anway by madspin, but just to make the cards cleaner.